### PR TITLE
Add the official logo and favicon

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add the official logo and favicon

## QA
- Run `./run`
- Open http://0.0.0.0:8026
- Check the logo in the navigation is multipass
- Check the favicon is multipass

Fixes https://github.com/canonical-web-and-design/multipass.run/issues/23

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div class="p-navigation__banner">
       <div class="p-navigation__logo">
         <a class="p-navigation__link" href="/">
-          <h4 class="p-navigation__image u-no-margin--bottom">Multipass</h4>
+          <img src="https://assets.ubuntu.com/v1/b96a2a65-Multipass+logo_rgb.svg" class="p-navigation__image" alt="Multipass">
```
Comment (barrymcgee) [inline _includes/navigation.html]:
```suggestion
          <img src="https://assets.ubuntu.com/v1/b96a2a65-Multipass+logo_rgb.svg" class="p-navigation__image" alt="Multipass logo">
```

**Comment by spencerbygraves** [comment]
@waynecrosby are you happy with the logo appearing like this in the hero section?

image

**Comment by anthonydillon** [comment]
@waynecrosby thanks, looks much better. The gradient is a flat image. @spencerbygraves could you provide a new background image?

**Comment by spencerbygraves** [comment]
@anthonydillon https://assets.ubuntu.com/v1/95cf2f74-multipass-hdr.svg

**Comment by waynecrosby** [comment]
Looks much better Ant thanks! Sorry to be a pain but I have noticed that the logo font is a bit heavy, could you please replace it with the one below:
https://assets.ubuntu.com/v1/d71f049c-Multipass+logo_thin_rgb_wht.svg

**Comment by anthonydillon** [comment]
@waynecrosby thanks, could you strip the whitespace around the logo please

**Comment by anthonydillon** [comment]
@spencerbygraves the new background has no colour. It appears its just the suru polygons with fill of `#fff`

**Comment by waynecrosby** [comment]
Logo looks much better, thanks!

**Comment by spencerbygraves** [comment]
@anthonydillon ok, try this one instead - https://assets.ubuntu.com/v1/84613674-multipass-hdr-02.jpg

**Comment by anthonydillon** [comment]
@spencerbygraves @waynecrosby done, let me know

**Comment by spencerbygraves** [comment]
@anthonydillon looks good to me 👍

**Pull request merged**